apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatView.kt

```diff
@@ -1497,6 +1504,108 @@ private fun TopEndFloatingButton(
   }
 }
 
+@Composable
+private fun FloatingDate(
+  modifier: Modifier,
+  listState: LazyListState,
+) {
+  var nearBottomIndex by remember { mutableStateOf(-1) }
```

Comment:
if you put it here you'll notice that recompositions happen every scroll offset changes, this shouldn't happen at all
```kotlin
SideEffect { println("LALAL RELOAD") }
```